tests/src/ObjectMapping.h

```diff
@@ -0,0 +1,47 @@
+#ifndef K4EDM4HEP2LCIOCONV_TEST_OBJECTMAPPINGS_H
+#define K4EDM4HEP2LCIOCONV_TEST_OBJECTMAPPINGS_H
+
+#include <unordered_map>
+
+namespace EVENT {
+  class Track;
+  class TrackerHit;
+  class TrackerHitPlane;
+  class SimTrackerHit;
+  class CalorimeterHit;
+  class RawCalorimeterHit;
+  class SimCalorimeterHit;
+  class TPCHit;
+  class Cluster;
+  class Vertex;
+  class ReconstructedParticle;
+  class MCParticle;
+  class LCEvent;
+} // namespace EVENT
+
+namespace podio {
+  class ObjectID;
+  class Frame;
+} // namespace podio
```

Comment:
Yes, mostly compile times a also habit of forward declaring as much as possible to reduce the things that need to be re-compiled simply because a (dependency) header changed, rather than just rebuilding the translation unit and re-linking it. I didn't do any measurements on the actual impact though.